account_payment_blocking/model/payment_order_create.py

```diff
@@ -27,8 +27,8 @@ class payment_order_create(orm.TransientModel):
     _inherit = 'payment.order.create'
 
     def extend_payment_order_domain(
-            self, cr, uid, payment_order, domain, context=None):
+            self, cr, uid, ids, payment_order, domain, context=None):
```

Comment:
You can migrate only these files, not the entire module. Old and new API can coexist, even in the same module or file